examples/hello-world/hello-indexer/src/lib.rs

```diff
@@ -27,8 +27,9 @@
 //! ```
 
 extern crate alloc;
-use fuel_indexer_macros::indexer;
-use fuel_indexer_plugin::prelude::*;
+use fuel_indexer_utils::macros::indexer;
+use fuel_indexer_utils::prelude::*;
+use fuel_indexer_utils::utilities::*;
```

Comment:
Could we re-export `fuel_indexer_utils::utilities::*` via `fuel_indexer_utils::prelude::*` here? And maybe `fuel_indexer_utils::macros::indexer` as well?

So ideally my import would be:

```rust
use fuel_indexer_utils::prelude::*
```

And that ☝🏽 would give me all utils, `#[indexer]` and whatever else is exported in `fuel_indexer_utils::prelude::*`

Reply:
Yep, I can do that.